src/ServiceControl.Audit/Auditing/FailedAuditImport.cs

```diff
@@ -1,10 +1,8 @@
 ﻿namespace ServiceControl.Audit.Auditing
 {
-    using System;
-
     public class FailedAuditImport
     {
-        public Guid Id { get; set; }
+        public string Id { get; set; }
```

Comment:
RavenDb 5 generates non-guid ids by default. This is how it was [handled in the prior upgrade branch](https://github.com/Particular/ServiceControl/blob/update_raven_to_5/src/ServiceControl.Audit/Auditing/FailedAuditImport.cs). We need to verify that RavenDb 3.5 handles this change.